src/content/en/updates/2017/06/headless-karma-mocha-chai.md

```diff
@@ -0,0 +1,137 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: "Testing on Headless Chrome with Karma/Mocha/Chai"
+
+{# wf_updated_on: 2017-06-13 #}
+{# wf_published_on: 2017-06-13 #}
+
+{# wf_tags: headless,testing,karma,mocha,chai #}
+{# wf_featured_image: /web/updates/images/generic/headless-chrome.png #}
+{# wf_featured_snippet: How to setup Karma, Mocha, and Chai to run automated tests on Headless Chrome. #}
+
+# Testing on Headless Chrome with Karma/Mocha/Chai {: .page-title }
+
+{% include "web/_shared/contributors/ericbidelman.html" %}
+
+Karma, Mocha, Chai, Headless Chrome, oh my!
+
+If you want to run automated tests using Headless Chrome, look no further! This article will get you
+all setup using Karma as a runner and Mocha/Chai for authoring tests.
```

Comment:
s/setup/set up/

"setup" is the noun we use when talking about a certain configuration, "set up" is the verb when we're talking about the act of creating the configuration